README.md

```diff
@@ -26,7 +26,7 @@ This needs to be done only once either while creating a project or enabling code
 
 ```
 composer require wunderio/code-quality --dev
-cp vendor/wunderio/code-quality/grumphp.yml.dist ./grumphp.yml
+cp vendor/wunderio/code-quality/config/grumphp.yml ./grumphp.yml
```

Comment:
So there is config folder and similar config is also under root folder. Could be confusing. I'm not sure what the difference is.

Reply:
Well we are using some configurations to test tool itself. Since run on paths and some other configuration differs from defaults then all default configs go under config folder.